feat(nuxt): Improve Nuxt server-side setup docs (#12102)
getsentry/sentry-docs: docs/platforms/javascript/guides/nuxt/install/limited-server-tracing.mdx

---

> title: Limited Server Tracing

Reviewer: Not sure, do we need to add a redirect for this renamed page? 🤔 Not sure how many links to the /top-level-import path are reasonably out there...?

Author: It was online for quite some time so maybe not a bad idea

sidebar_order: 2

> description: "Learn about how the Nuxt SDK can be set up with limited server tracing by adding a top-level import to the build output."

Reviewer 2: ```suggestion
description: "Learn how to set up the Nuxt SDK with limited server tracing by adding a top-level import to the build output."
```

---

## Understanding Limited Server Tracing

> Sentry needs to be initialized before the application starts.

Reviewer: ```suggestion
Sentry needs to be initialized before the import graph is resolved by Node.
```

> If the default way of adding an <PlatformLink to="/install/cli-import">`--import` CLI flag</PlatformLink> flag does not work for you,

Reviewer 2: ```suggestion
If the default way of adding an <PlatformLink to="/install/cli-import">`--import` CLI flag</PlatformLink> flag doesn't work for you,
```

> set up the SDK for adding a top-level `import`.

Reviewer 2: ```suggestion
enable the SDK to add a top-level `import`.
```

> The automatically added top-level `import` will import the Sentry server-side config at the top of the Nuxt server entry file.

Reviewer 2: ```suggestion
The automatically added top-level `import` will then import the Sentry server-side config to the top of the Nuxt server entry file.
```

```

By default, the SDK will add the Sentry server config to the build output (typically, `.output/server/sentry.server.config.mjs`).

> The SDK then automatically imports this file at the top of the Nitro server entry in the build output.

Reviewer 2: ```suggestion
The SDK will then automatically import this file to the top of the Nitro server entry in the build output.
```
